pkg/kademlia/replacement_cache.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package kademlia
+
+import (
+	proto "storj.io/storj/protos/overlay"
+	"storj.io/storj/storage"
+)
+
+func (rt *RoutingTable) updateReplacementCache(kadBucketID storage.Key, nodes []*proto.Node) {
+	bucketID := string(kadBucketID)
+	rt.replacementCache[bucketID] = nodes
+}
+
+func (rt *RoutingTable) getReplacementCacheBucket(kadBucketID storage.Key) []*proto.Node {
+	return rt.replacementCache[string(kadBucketID)]
+}
```

Comment:
these two functions above are pretty short. i wonder if they are actually making the code more readable? i don't think i mind seeing rt.replacementCache[string(kadBucketID)] instead of rt.getReplacementCacheBucket(kadBucketID).